.gitignore

```diff
@@ -65,6 +65,9 @@ DerivedData
 *.xcuserstate
 project.xcworkspace
 
+# Xcode/iOS
+ios/
```

Comment:
I don't understand this.

Reply:
When you want to run the app on an iPhone simulation, it generates an ios/ folder with the Xcode project. I thought it would be unnecessary to push up but it already looks like it's in the repo. Should I remove it?

Reply:
I'd remove the gitignore change - its not relevant to the change.